app/models/concerns/foreman_templates/template_extensions.rb

```diff
@@ -0,0 +1,11 @@
+require 'shellwords'
+
+module ForemanTemplates
+  module TemplateExtensions
+    extend ActiveSupport::Concern
+
+    def template_file
+      Shellwords.escape(name.downcase.tr(' /', '_') + '.erb')
```

Comment:
Is the escaping necessary?

Reply:
I think it might be needed:

```ruby
"a/b'c d".downcase.tr(' /', '_')
#=> "a_b'c_d"

Shellwords.escape "a/b'c d".downcase.tr(' /', '_')
#=> "a_b\\'c_d"
```

Reply:
Does it matter that there's a quote in the name? Afaik we shouldn't be passing the name to a shell anywhere